e2e/helpers/index.js

```diff
@@ -26,9 +27,9 @@ function assertVisuals({browser, id}) {
   }
 }
 
+// Disables the visual tests for now while waiting for the baseline approved
 module.exports = {
   getUrl,
   goToUrl,
   formatFileName,
-  assertVisuals,
```

Comment:
if we don't export, consuming code will fail
try changing this flag to false

<img width="374" alt="image" src="https://user-images.githubusercontent.com/1851469/47516676-ef579f80-d83a-11e8-87b9-e164b78667ca.png">

Reply:
Good catch! Updated.